tools/image-builder/build_bond.zsh

```diff
@@ -0,0 +1,82 @@
+#!/bin/zsh
+
+set -eux
+setopt shwordsplit
```

Comment:
Please add a comment explaining in what way/why we're depending on this behavior.